.github/workflows/android-release-artifacts.yml

```diff
@@ -89,11 +90,22 @@ jobs:
           sed -i "s/\(coordinates(\"org.pytorch\", \"executorch-android\", \"\)\([0-9]\+.[0-9]\+.[0-9]\+\)\(\")\)/\1$VERSION\3/" extension/android/executorch_android/build.gradle
         fi
 
+        GRADLE_ARGS=""
+
         FLAVOR="${{ inputs.flavor }}"
         if [[ "$FLAVOR" == "vulkan+xnnpack" || -z "$FLAVOR" ]]; then
           export EXECUTORCH_BUILD_VULKAN=ON
         fi
 
+        if [[ "$FLAVOR" == "qnn" ]]; then
+          PYTHON_EXECUTABLE=python bash .ci/scripts/setup-qnn-deps.sh
+          PYTHON_EXECUTABLE=python bash .ci/scripts/build-qnn-sdk.sh
+          export EXECUTORCH_BUILD_QNN=ON
+          export QNN_SDK_ROOT=/tmp/qnn/2.28.0.241029
```

Comment:
Can we try to reduce the hardcode version and refer to this https://github.com/pytorch/executorch/blob/main/backends/qualcomm/scripts/qnn_config.sh ? If it needs to refactor a bit that's fine